### PR TITLE
python-modernize . -w --future-unicode

### PR DESCRIPTION
**Pull request by nickpell**
@dimagi/py3

### CODE REVIEW

File: doc/couchdbkit.org/buildweb.py [hunk 1]
```diff
@@ -154,16 +155,16 @@ def generate_rss(self):
...
-            xml += u'<priority>0.5</priority>'
-            xml += u'</url>'
-        xml += u'</urlset>'
+            xml += '<url>'
+            xml += '<loc>%s</loc>' % page.url
+            xml += '<lastmod>%s</lastmod>' % rfc3339_date(page.headers['published'])
+            xml += '<changefreq>daily</changefreq>'
+            xml += '<priority>0.5</priority>'
+            xml += '</url>'
+        xml += '</urlset>'
```
Comment (esoergel) [inline doc/couchdbkit.org/buildweb.py]: wat

**Comment by emord** [comment]
fwiw I had been holding off on this because we're still reliant on restkit for a lot of things and there are quite a few tests that are reliant on testing bytes vs unicode. Although if both these and HQ tests pass on this, it's probably safe to go.

File: doc/couchdbkit.org/buildweb.py (hunk 1 above)
Comment (emord) [reply doc/couchdbkit.org/buildweb.py]: Yeah. we should probably clean up a lot of the random crap in this repo. The upstream project is no longer maintained and I think we long gave up on PRing our jsonobject fork

**Comment by nickpell** [comment]
It looks like restkit is not playing well in tests - I'm going to hold off until we're off restkit

**Pull request closed without merging**